wren-ui/src/apollo/server/adaptors/ibisAdaptor.ts

```diff
@@ -44,7 +70,7 @@ export interface ValidationResponse {
 
 export interface IbisQueryOptions {
   dataSource: DataSourceName;
-  connectionInfo: IbisBigQueryConnectionInfo | IbisPostgresConnectionInfo;
+  connectionInfo: WREN_AI_CONNECTION_INFO;
```

Comment:
I'm thinking which make more sense to use as type here, `WREN_AI_CONNECTION_INFO` or `IbisConnectionInfo`.

Reply:
Conclusion: keep it for now.

Will refactor to sth like following in the next release
```js
export const DataSources = {
    [DataSourceName.MYSQL]: {
        config: {
            ...
        },
        sensitiveProperties: ["credentials"],
        toIbisAdaptorConnectionInfo: () => {...}
    }
};
```